client/packages/common/src/ui/components/charts/ValueBar/ValueBar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import { Box, Tooltip, Typography } from '@openmsupply-client/common';
+import { useFormatNumber } from '@common/intl';
+
+const MIN_FLEX_BASIS_TO_SHOW_LABEL = 10;
+const MIN_FLEX_BASIS_TO_SHOW_VALUE = 5;
+
+const Divider = () => (
+  <Box sx={{ backgroundColor: 'gray.dark', width: '1px', height: '45px' }} />
+);
+
+interface ValueBarProps {
+  value: number;
+  total: number;
+  label: string;
+  colour: string;
+  startDivider?: boolean;
+  endDivider?: boolean;
+}
+
+export const ValueBar: FC<ValueBarProps> = ({
+  value,
+  total,
+  label,
+  colour,
+  startDivider = false,
+  endDivider = true,
+}) => {
+  const formatNumber = useFormatNumber();
+  if (value === 0) return null;
```

Comment:
The previous behaviour rendered the start divider regardless - something like this would do it:
```suggestion
  if (value === 0) return startDivider ? <Divider /> : null;
```

Reply:
That makes sense.  Commit submitted.  Cheers.

Reply:
I've just noticed on one of the "ResponseStoreStats" example, the Dividers don't show at all (and labels are ugly).
They are there, they just have a width of 0.  It has to do with the way 'requestedPercent' is calculated for the width of the div.  Not sure if that should be tackled here in this PR or not.

![Screenshot 2023-03-30 122659](https://user-images.githubusercontent.com/17306152/228690087-7ac93312-07af-40f6-bae2-a7fb617b5ba7.png)